src/jaxsim/api/link.py

```diff
@@ -54,9 +54,7 @@ def idx_to_name(model: js.model.JaxSimModel, *, link_index: jtp.IntLike) -> str:
     """
 
     exceptions.raise_value_error_if(
-        condition=jnp.array(
-            [link_index < 0, link_index >= model.number_of_links()]
-        ).any(),
+        condition=link_index < 0,
```

Comment:
Why the case `link_index >= model.number_of_links()` has been removed?

Reply:
`model.kin_dyn_parameters.link_names` is a `Static[tuple[str, ...]]`, so if the link index is out of bounds, it will raise an `IndexError` anyway

Reply:
Ok thanks!